src/logger.ts

```diff
@@ -0,0 +1,16 @@
+import * as winston from 'winston';
+
+export const logger = winston.createLogger({
```

Comment:
Any reason in particular for sticking with winston vs https://github.com/pinojs/pino (the one that Fastify uses)?

Reply:
Oh neat, I just used winston because of the API. I'll switch to pino.

Reply:
Done